### PR TITLE
#75 [FE] Setup Integration test setup 

### PR DESCRIPTION
**Pull request by AliceHuang1027**
1. create a route in **server.js** that goes to / and just renders welcome in **welcome.ejs** 
2. create an integration test in **test/integration/welcome.test.js** to start stop server and sends a request to / to match snapshot.
3.Setup view engine: install ejs and app.set('view engine','ejs') 
4. changed part of server.test.js to test the app.set and app.get

Confusion: http://127.0.0.1 vs localhost  (since the former did not pass the test, the latter passed)
image is the former case (failed the test)
Screen Shot 2020-06-09 at 5 35 56 AM

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ const startServer = async (portNumber) => {
   await dbModule.start()
   return new Promise((resolve, reject) => {
     app = express()
+    app.set('view engine','ejs')
+    app.use(express.json())
+    app.get('/',(req,res)=>{
+      const user ={name:'donnell'}
```
Comment (greenc123) [inline src/server.js:19]: Who is donnell?

Comment (greenc123) [reply src/server.js:19]: Was this `user` object for testing purposes? It doesn't seem like it should be in a commit or code review file.

Comment (AliceHuang1027) [reply src/server.js:19]: Yea I was thinking whether to just take it out. However in that case we don’t need to use ejs

Comment (AliceHuang1027) [reply src/server.js:19]: > Who is donnell?

It’s one of my favorite comedians-Donnell Rawlings!

**Pull request closed without merging**